Sources/JavaKitConfigurationShared/GenerationMode.swift

```diff
@@ -76,3 +76,29 @@ extension JExtractMinimumAccessLevelMode {
     .public
   }
 }
+
+
+/// Configures how memory should be managed by the user
+public enum JExtractMemoryManagementMode: String, Codable {
+  /// Force users to provide an explicit `SwiftArena` to all calls that require them.
+  case forceExplicit
```

Comment:
Maybe just `explicit`
```suggestion
  case explicit
```